src/ui/components/counter-list.component.ts

```
@@ -0,0 +1,229 @@
+        if (!this.listHostDiv || !this.listHostDiv.isConnected) {
+            this.element.innerHTML = '';
+            this.listHostDiv = document.createElement('div');
+            this.listHostDiv.id = 'counter-list'; // From original HTML
+            this.appendChild(this.listHostDiv);
+        }
+    }
+
+    private renderFullList(): void {
+        if (this.counters.length === 0) {
+            this.element.innerHTML = '<p>Noch keine Tresen für diesen Standort erfasst.</p>';
+            if (this.listHostDiv && this.listHostDiv.isConnected) this.listHostDiv.remove();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)